pkg/util/files_test.go

```diff
@@ -0,0 +1,23 @@
+package util
+
+import (
+	"github.com/stretchr/testify/assert"
+	"testing"
+)
+
+func TestContentTypeForFile(t *testing.T) {
```

Comment:
this test is completely platform specific (not even OS specific)

If the code is relying on this to actually be that then it should really use a map.

from the code that eventually is called

```
// TypeByExtension returns the MIME type associated with the file extension ext.
// The extension ext should begin with a leading dot, as in ".html".
// When ext has no associated type, TypeByExtension returns "".
//
// Extensions are looked up first case-sensitively, then case-insensitively.
//
// The built-in table is small but on unix it is augmented by the local
// system's mime.types file(s) if available under one or more of these
// names:
//
//   /etc/mime.types
//   /etc/apache2/mime.types
//   /etc/apache/mime.types
//
// On Windows, MIME types are extracted from the registry.
//
// Text types have the charset parameter set to "utf-8" by default.
```

Reply:
I don;t know what the actual requirements are for the code here,  if you just need something then this is ok - but then no need to test that `something` is `X`.
if you need something for specific files - then could we just use a simple map with a lookup?